lib/livebook/hubs.ex

```diff
@@ -180,4 +213,12 @@ defmodule Livebook.Hubs do
       [] -> nil
     end
   end
+
+  defp hub_connected?(hub) do
+    if connected_hub = get_connected_hub(hub) do
+      GenServer.call(connected_hub.pid, :connected?)
```

Comment:
Instead of doing this, we should add a new callback to the Hub behaviour. Then you should be able to do this on EnterpriseClient:

    try do
      GenServer.call({:via, Registry, {HubsRegistry, hub.id}}, :connected?)
    catch
      :exit, _ -> false
    end